eve/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: 0.2
+version: 0.3
```

Comment:
You can leave this to 0.2
```suggestion
version: 0.2
```
I don't think 0.3 exists in eve :D

Reply:
perfect, done, I thought it merely was a version identifier of the yml file